app/controllers/api/v1/spots_controller.rb

```diff
@@ -1,7 +1,24 @@
 class Api::V1::SpotsController < ApplicationController
 
   def update
+    classifiers = ImageClassifier.create_classifiers(get_image_url)
+    classifiers.each do |classifier|
+      if classifier.classification == 'vehicle' && classifier.score > 0.75
```

Comment:
@sallymacnicholas do you know of a good way to refactor this? I wanted to put it in the ImageClassifier model, but the return statement is ejecting me from the loop.